lib/mastodon/version.rb

```diff
@@ -48,5 +48,9 @@ def source_url
         source_base_url
       end
     end
+
+    def user_agent
+      @user_agent ||= "#{HTTP::Request::USER_AGENT} (Mastodon/#{Version}; +http#{Rails.configuration.x.use_https ? 's' : ''}://#{Rails.configuration.x.local_domain}/)"
```

Comment:
Why not use `root_url`? Also, this is going to link to e.g., https://sitedethib.com instead of https://social.sitedethib.com, (`WEB_DOMAIN` vs. `LOCAL_DOMAIN`) which isn't a huge deal I guess, but it still feels a bit weird.

Reply:
`root_url` requires `RoutingHelper` but not loaded in this scope.
I will rewrite to use WEB_DOMAIN...